doc/en/user/source/production/troubleshooting.rst

```diff
@@ -47,198 +50,199 @@ All tools work against a live Java Virtual Machine, the one running GeoServer in
-	 - java.lang.Thread.run() @bci=11, line=619 (Interpreted frame)
-	
-	Locked ownable synchronizers:
-	    - None
-	
-	Thread 16267: (state = BLOCKED)
-	 - java.lang.Object.wait(long) @bci=0 (Interpreted frame)
-	 - java.lang.Object.wait() @bci=2, line=485 (Interpreted frame)
-	 - org.apache.tomcat.util.threads.ThreadPool$ControlRunnable.run() @bci=26, line=662 (Interpreted frame)
-	 - java.lang.Thread.run() @bci=11, line=619 (Interpreted frame)
-	
-	Locked ownable synchronizers:
-	    - None
-	    
-	...
+   Deadlock Detection:
+   
+   No deadlocks found.
+   
+   Thread 16269: (state = BLOCKED)
+    - java.lang.Object.wait(long) @bci=0 (Interpreted frame)
+    - org.apache.tomcat.util.threads.ThreadPool$MonitorRunnable.run() @bci=10, line=565 (Interpreted frame)
+    - java.lang.Thread.run() @bci=11, line=619 (Interpreted frame)
+   
+   Locked ownable synchronizers:
+       - None
+   
+   Thread 16268: (state = IN_NATIVE)
+    - java.net.PlainSocketImpl.socketAccept(java.net.SocketImpl) @bci=0 (Interpreted frame)
+    - java.net.PlainSocketImpl.accept(java.net.SocketImpl) @bci=7, line=390 (Interpreted frame)
+    - java.net.ServerSocket.implAccept(java.net.Socket) @bci=60, line=453 (Interpreted frame)
+    - java.net.ServerSocket.accept() @bci=48, line=421 (Interpreted frame)
+    - org.apache.jk.common.ChannelSocket.accept(org.apache.jk.core.MsgContext) @bci=46, line=306 (Interpreted frame)
+    - org.apache.jk.common.ChannelSocket.acceptConnections() @bci=72, line=660 (Interpreted frame)
+    - org.apache.jk.common.ChannelSocket$SocketAcceptor.runIt(java.lang.Object[]) @bci=4, line=870 (Interpreted frame)
+    - org.apache.tomcat.util.threads.ThreadPool$ControlRunnable.run() @bci=167, line=690 (Interpreted frame)
+    - java.lang.Thread.run() @bci=11, line=619 (Interpreted frame)
+   
+   Locked ownable synchronizers:
+       - None
+   
+   Thread 16267: (state = BLOCKED)
+    - java.lang.Object.wait(long) @bci=0 (Interpreted frame)
+    - java.lang.Object.wait() @bci=2, line=485 (Interpreted frame)
+    - org.apache.tomcat.util.threads.ThreadPool$ControlRunnable.run() @bci=26, line=662 (Interpreted frame)
+    - java.lang.Thread.run() @bci=11, line=619 (Interpreted frame)
+   
+   Locked ownable synchronizers:
+       - None
+       
+   ...
   
   
 jmap
 ````
 
-`jmap <http://java.sun.com/javase/6/docs/technotes/tools/share/jmap.html>`_ is a tool to gather information about the a Java virtual machine. 
+`jmap <http://java.sun.com/javase/6/docs/technotes/tools/share/jmap.html>`__ is a tool to gather information about the a Java virtual machine. 
 It can be used in a few interesting ways.
 
 By running it without arguments (past the pid of the JVM) it will print out a **dump of the native libraries used by the JVM**. This can come in handy when one wants to double check GeoServer is actually using a certain version of a native library (e.g., GDAL)::
```

Comment:
```suggestion
By running it without arguments (other than the process id of the JVM) it will print out a **dump of the native libraries used by the JVM**. This can come in handy when one wants to double check GeoServer is actually using a certain version of a native library (e.g., GDAL)::
```